src/main/resources/application-test.yml

```diff
@@ -0,0 +1,46 @@
+liquibase:
+  url: jdbc:postgresql://localhost:15432/postgres
+
+spring:
+  datasource:
+    url: jdbc:postgresql://localhost:15432/postgres
+
+survey-svc:
+  connection-config:
+    port: 18002
+
+action-svc:
+  connection-config:
+    port: 38151
+
+redisson-config:
+  address: localhost:17379
+
+rabbitmq:
+  host: localhost
+  port: 35672
+
+data-grid:
+  address: localhost:17379
+
+schedules:
+  validation-schedule-delay-milli-seconds: 5000
```

Comment:
Are these delays depended on in the tests? Might be worth putting to 1s if they are to speed them up